tests/test_updater_root_rotation_integration.py

```diff
@@ -413,8 +481,103 @@ def test_root_rotation_unmet_threshold(self):
 
     # The following refresh should fail because root must be signed by the
     # previous self.role_keys['root'] key, which wasn't loaded.
-    self.assertRaises(tuf.exceptions.NoWorkingMirrorError,
-        self.repository_updater.refresh)
+    with self.assertRaises(tuf.exceptions.NoWorkingMirrorError) as cm:
+      self.repository_updater.refresh()
+
+    for mirror_url, mirror_error in six.iteritems(cm.exception.mirror_errors):
+      # Verify that '2.root.json' is the culprit.
+      self.assertTrue(mirror_url.endswith('/3.root.json'))
```

Comment:
Now that the code is simpler it's easy to see an issue with the comment :)
Could even drop the comment: code seems fairly obvious to me...

Reply:
🤦 thank you for the diligent review. Fixed in 6eaf194